_data/projects/Artistic-Neural-Style-Transfer.yml

```diff
@@ -0,0 +1,11 @@
+name: Artistic Neural Style Transfer
+desc: Artistic Neural Style Transfer helps to transfer the style of your favourite artist style image to any content image.
+site: https://github.com/riya-17/Neural-Artistic-Style-Transfer
+tags:
+# Note: these are tags categorizing the project, not issue labels.
+- Tensorflow
+- python
+- numpy
+upforgrabs:
+  name: good first issue
+link: https://github.com/riya-17/Neural-Artistic-Style-Transfer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

Comment:
Please correct the indentation for link and add your label link instead of issue link